src/logger/constants.ts

```diff
@@ -0,0 +1,14 @@
+import pino from "pino"
+
+// eslint-disable-next-line import/prefer-default-export
+export const PINO_OPTIONS = {
+  // NOTE: DO NOT format time in-process
+  // as this will affect perf of the logger.
+  // See here: https://getpino.io/#/docs/api?id=timestamp-boolean-function
+  timestamp: pino.stdTimeFunctions.isoTime,
```

Comment:
Just a note on that: using `pino.stdTimeFunctions.isoTime` **is** formatting time in process 😑😅  . The "cheap" option is to leave the default `timestamp:true` to get the plain integer timestamp and "out of process" means you transform to human readable string when you use/process logs.

[Implementation in pino for reference](https://github.com/pinojs/pino/blob/dbb394fc8d4bafad26814e39ee9fbfab1f59fe70/lib/time.js#L9)

Reply:
wow i got scammed, i thought their docs on the `timestamp` there would be out of process. tbh, pino uses a worker process to pipe logs over to cw, so it depends on what their "process" here refers to (main thread or worker thread). 

in any case, no harm just removing this - if we're relying on log integration, the trace itself should have a timestamp and there's already 2 hops for our log (console -> log group, log group -> dd) so i think less latency the better